plugins/kotlin-dataframe/testData/box/readJsonStr_datarow.kt

```diff
@@ -0,0 +1,13 @@
+import org.jetbrains.kotlinx.dataframe.*
+import org.jetbrains.kotlinx.dataframe.annotations.*
+import org.jetbrains.kotlinx.dataframe.api.*
+import org.jetbrains.kotlinx.dataframe.io.*
+
+const val text = """{"a":"abc", "b":1}"""
+
+fun box(): String {
+    val row = DataRow.readJsonStr(text)
+    row.a
```

Comment:
This would be so awesome for an ordinary Map :o I wonder if we could have a plugin which just does this for Maps. It's like js/python but safe :P